templates/Segment_Image_Data.ipynb

```diff
@@ -358,7 +358,29 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "### Afterwards, we can generate expression matrices from the labeling + imaging data"
+    "### Afterwards, we can generate expression matrices from the labeling + imaging data\n",
```

Comment:
I think this is a fairly niche use case, at least for now. Did Tyler want it for some reason? I think it's just going to add confusion if we include it in the main notebook

Reply:
Hadeesha wanted the positive pixel extraction.  Makes extra sense for exotic (eg. brain) panels where some markers might not stain well (making the intensity differences between channels less directly comparable).  But I do agree that it looks kinda out of place in the main notebook (at least the way I wrote it this pass).

Reply:
For sure. If he has questions you can point him to the documentation to see how to get it to work